tendrl/monitoring_integration/sync/__init__.py

```diff
@@ -44,6 +47,8 @@ def run(self):
                         )
                         raise ex
                 except etcd.EtcdKeyNotFound as ex:
+                    # Before cluster import sync_interval is not populated
+                    time.sleep(DEFAULT_SLEEP)
```

Comment:
monitoring integration uses sync_intervel from gluster NS, but before import cluster namespace is not available so it consumes lot of cpu

Reply:
Consumes lot of course as there is no sleep at all while sync loop and actually thread in entering a infinite while loop without any sleep. So it's good to default for sleep if cluster not imported yet.